### PR TITLE
Contributions

### PR DESCRIPTION
**Pull request by PawelKiedrzynski**
Check my pull request.

### CODE REVIEW

File: src/main/java/pl/net/rogala/eventy/controller/EventController.java [hunk 1]
```diff
@@ -43,33 +46,41 @@ public String showSingleEvent(@PathVariable String eventId, Authentication authe
...
-
+//        model.addAttribute("showUsers", eventService.showAllUsersAssignedToEvent(Long.parseLong(eventId)));
 
         return "event/showSingleEvent";
     }
+
+    @PostMapping("/event/{id}/addUserToEvent")
+    public String handleNewUserAssignedToEvent(@PathVariable String id,@RequestParam String eventId, Authentication authentication) {
+        eventService.assignedUseToEvent(Long.parseLong(id), authentication.getName());
+        return "redirect:/event/" + eventId;
```
Comment (szymanskimichal90) [inline src/main/java/pl/net/rogala/eventy/controller/EventController.java]: I think you should remove @RequestParam String eventId and you should replace in return statement eventId by id

File: src/main/java/pl/net/rogala/eventy/configuration/BasicConfiguration.java [hunk 2]
```diff
@@ -7,7 +7,9 @@
 
 @Configuration
 public class BasicConfiguration {
-
+//ola@nowak.pl
```
Comment (owczarekblazej) [inline src/main/java/pl/net/rogala/eventy/configuration/BasicConfiguration.java]: Suggest: delete this comment

Comment (owczarekblazej) [inline src/main/java/pl/net/rogala/eventy/configuration/BasicConfiguration.java]:
```suggestion 

```

**Review by owczarekblazej — COMMENTED** [review]
Delete comments ;-)

**Review by szymanskimichal90 — APPROVED** [review]
It should works

**Pull request merged**